src/Contract.php

```diff
@@ -72,17 +78,22 @@ public function toPdf(HtmlConverterInterface $htmlConverter, array $data = [])
     /**
      * Generates file and sends a SignRequest
      * @param DocumentSignInterface $documentSign
+     * @return bool|void
      */
     public function sign(DocumentSignInterface $documentSign)
     {
-        if (!$this->file || !config('mvc-contracts.document_sign.enabled')) {
+        $config = config('mvc-contracts.document_sign');
+
+        if (!$this->file || !$config['enabled']) {
             return;
         }
 
         try {
             $document = $documentSign->create(storage_path($this->file), Str::replaceFirst('contracts/', '', $this->file));
             $this->external_id = $document['id'];
-            $documentSign->signer($this->user->email)->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
+            $redirectUrl = str_replace('{id}', $this->id, $config['return']);
```

Comment:
This should only be done when config "return" is configured. Projects might not have redirect urls. use Arr::has